source/mainnet/net/mobile-wallet/add-baker-mw.rst

```diff
@@ -0,0 +1,70 @@
+
+    .. image:: ../images/mobile-wallet/add-baker-amt-mw.png
+
+.. Note::
+
+    There is a minimum amount to stake to become a baker. To find out what the minimum stake is, you can run the :ref:`consensus show-chain-parameters transaction in concordium-client<consensus show-chain-parameters>`.
+
+.. Warning::
+
+   Do not stake all of your funds or you will not have enough to cover transaction fees for unstaking or other transactions.
+
+4. Choose whether you want to open a :ref:`baker pool<glossary-baker-pool>` or keep it closed. By opening a baker pool, others can delegate stake to your baker, thus increasing the chance that you are selected to bake a block and earn rewards. If you have a baker pool with delegators, the delegators also earn rewards when you bake blocks. Bakers are also paid a commission by the delegators for baking on their behalf. You can choose **Close for delegation** if you do not wish to run a baker pool.
+
+    .. image:: ../images/mobile-wallet/add-baker-pool-mw.png
+
+5. If you open a baker pool, you can optionally enter a URL with information about your baker to give delegators more information about your baker pool to help them research baker pools. This information is not shared in the case of a closed baker.
+
+    .. image:: ../images/mobile-wallet/add-baker-pool-url-mw.png
+
+6. You have to export the baker credentials so that you can start the node with the baker keys. Tap **Export baker keys** and navigate to the place on your device where you want to save the file. If you're running Windows make sure that **All Files** is selected in **Save as type**. Give the file a name and the extension .json, and then click **Save** and navigate to the place on your computer where you want to save the file.
+
+    .. image:: ../images/mobile-wallet/add-baker-export-keys-mw.png
+
+.. Warning::
+
+    This is the only time that you can export the credentials. If you're going to transfer the baker keys to someone else, make sure to do so through a secure channel. Generate new keys if you believe the keys have been compromised or lost.
+
+7. Once you have saved the keys, you see an overview screen of the add baker transaction. Review the information then tap **Submit baker transaction**.
+
+    .. image:: ../images/mobile-wallet/add-baker-submit-mw.png
+
+8. In the Mobile Wallet, you can see that the transaction has been submitted to the chain. Tap **Finish**. Once the transaction is approved, you see the active baker screen where you have the possibility to update baker settings.
+
+    .. image:: ../images/mobile-wallet/add-baker-finish-mw.png
+
+9. You need to import your baker keys file to your node in order to start baking. It is preferable to update them on the node as close to :ref:`pay day<glossary-pay-day>` as possible to prevent the node from being down as a baker for a longer time.
```

Comment:
```suggestion
9. You need to import your baker keys file to your node in order to start baking. It is preferable to update them on the node as close to the next :ref:`pay day<glossary-pay-day>` as possible to prevent the node from being down as a baker for a longer time.
```